doc/ranger.pod

```diff
@@ -527,8 +527,8 @@ also "da", "dr" and "dt" shortcuts equivalent to "ya", "yr" and "yt".)
 Paste the files which were previously copied or cut, like pressing Ctrl+V in
 modern GUI programs.
 
-Conflicts will be renamed by appending an _ to the file extension. To change
-this see the paste command.
+Conflicts will be renamed by appending an _ to the full file name. To change
+this and insert a _ before the file extension see commands.py.
```

Comment:
Please check elsewhere in the file, I think we use some sort of formatting for files. `F<file.ext>` maybe? Maybe we should also quote the underscores while we're at it? Look for other examples of single literal characters in the manual to check.

Reply:
fixed